cluster/charts/crossplane/templates/namespace-personas-clusterroles.yaml

```diff
@@ -0,0 +1,139 @@
+{{- if .Values.personas.deploy }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:stack-manager:ns:default:admin
+  labels:
+    rbac.crossplane.io/aggregate-to-crossplane-admin: "true"
+    rbac.crossplane.io/aggregate-to-namespace-default-admin: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "namespace"
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.crossplane.io/aggregate-to-namespace-admin: "true"
+rules:
+- apiGroups:
+  - cache.crossplane.io
+  - compute.crossplane.io
+  - database.crossplane.io
+  - kubernetes.crossplane.io
+  - core.crossplane.io
+  - storage.crossplane.io
+  - workload.crossplane.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - stacks.crossplane.io
+  resources:
+  - stackinstalls
+  - stacks
+  verbs:
+  - "*"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:stack-manager:ns:default:edit
+  labels:
+    rbac.crossplane.io/aggregate-to-namespace-admin: "true"
+    rbac.crossplane.io/aggregate-to-namespace-default-edit: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "namespace"
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.crossplane.io/aggregate-to-namespace-edit: "true"
+rules:
+- apiGroups:
+  - cache.crossplane.io
+  - compute.crossplane.io
+  - database.crossplane.io
+  - kubernetes.crossplane.io
+  - core.crossplane.io
+  - storage.crossplane.io
+  - workload.crossplane.io
+  resources:
+  - "*"
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - patch
+  - update
+- apiGroups:
+  - stacks.crossplane.io
+  resources:
+  - stackinstalls
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - "ConfigMaps"
+  - "Secrets"
+  verbs:
+  - get
+  - list
+  - watch
+  - create
+  - delete
+  - deletecollection
+  - patch
+  - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}:stack-manager:ns:default:view
+  labels:
+    rbac.crossplane.io/aggregate-to-namespace-edit: "true"
+    rbac.crossplane.io/aggregate-to-namespace-default-view: "true"
+    app: {{ template "name" . }}
+    chart: {{ template "chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    crossplane.io/scope: "namespace"
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      rbac.crossplane.io/aggregate-to-namespace-view: "true"
```

Comment:
I think this may cause problems as any namespace specific install will match here. Also if we have match labels we wouldn't define any rules statically.

```suggestion

```